src/containers/EditMarkupPage/EditMarkupPage.js

```diff
@@ -197,7 +239,11 @@ export class EditMarkupPage extends Component {
                 />
                 <Row justifyContent="end" alignItems="baseline">
                   <Link
-                    to={`/subject-matter/learning-resource/${draftId}/edit/${language}`}>
+                    to={
+                      state
+                        ? state.backUrl
+                        : `/subject-matter/learning-resource/${draftId}/edit/${language}`
```

Comment:
Pirk, men jeg elsker ikke å bruke `state` som variabelnavn bare fordi det er så lett for nabbs som meg å mixe det inn med komponenten sin state.

```suggestion
                      state?.backUrl ||
                      `/subject-matter/learning-resource/${draftId}/edit/${language}`
```